src/AndcultureCode.CSharp.Sitefinity.Conductors/Extensions/DynamicContentExtensions.cs

```diff
@@ -0,0 +1,116 @@
+﻿using AndcultureCode.CSharp.Sitefinity.Core.Attributes;
+using AndcultureCode.CSharp.Sitefinity.Core.Models.Content;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using Telerik.Sitefinity.DynamicModules;
+using Telerik.Sitefinity.DynamicModules.Model;
+using Telerik.Sitefinity.GenericContent.Model;
+using Telerik.Sitefinity.Model;
+using Telerik.Sitefinity.RelatedData;
+using Telerik.Sitefinity.Utilities.TypeConverters;
+
+namespace AndcultureCode.CSharp.Sitefinity.Conductors.Extensions
+{
+    public static class DynamicContentExtensions
+    {
+        public static DynamicContent SetDataItemProperties<T>(this T item, PropertyInfo[] properties, string providerName) where T : SitefinityContent
```

Comment:
I think we should add summary comments to the extension methods in this class describing what each of them do.